test/core/test_capture.js

```diff
@@ -30,88 +74,178 @@ test('Capture - headers', (t) => {
   });
 });
 
-test('Capture - JSON', (t) => {
-  const fn = path.resolve(__dirname, './scripts/captures.json');
```

Comment:
Should remove this and all other JSON scripts that have been inlined into tests if they're not used anywhere else.